tests/pipelines/test_pipelines_visual_question_answering.py

```diff
@@ -172,7 +179,35 @@ def test_large_model_pt_blip2(self):
         outputs = vqa_pipeline([{"image": image, "question": question}, {"image": image, "question": question}])
         self.assertEqual(outputs, [[{"answer": "two"}]] * 2)
 
+    @slow
+    def test_pt_beit3(self):
+        vqa_pipeline = pipeline(
+            "visual-question-answering",
+            model="Raghavan/beit3_base_patch16_480_vqa",
```

Comment:
Reminder to update this checkpoint before merging

Reply:
Sure.There is 22 variations of the model, Once approaved, I will convert every one of them .

Reply:
I am keeping this conversation open, just to remind myself to update the checkpoint and do the model conversions.